packages/blockchain/src/index.ts

```diff
@@ -105,6 +107,32 @@ export interface BlockchainOptions {
    * provided from the `common` will be used.
    */
   genesisBlock?: Block
+
+  /**
+   * If you are using a custom chain {@link Common}, pass the genesis state.
+   *
+   * Pattern 1 (with genesis state see {@link GenesisState} for format):
+   *
+   * ```javascript
+   * {
+   *   '0x0...01': '0x100', // For EoA
+   * }
+   * ```
+   *
+   * Pattern 2 (with complex genesis state, containing contract accounts and storage).
+   * Note that in {@link AccountState} there are two
+   * accepted types. This allows to easily insert accounts in the genesis state:
+   *
+   * A complex genesis state with Contract and EoA states would have the following format:
+   *
+   * ```javascript
+   * {
+   *   '0x0...01': '0x100', // For EoA
+   *   '0x0...02': ['0x1', '0xRUNTIME_BYTECODE', [[storageKey1, storageValue1], [storageKey2, storageValue2]]] // For contracts
+   * }
+   * ```
+   */
+  genesisState?: GenesisState
```

Comment:
Ok, just followed this along just to get some confirmation: so if one now would want to run a VM with/upon some custom genesis state, one would create a Blockchain object first with this genesis state and passes this into the VM along initialization. Seems a reasonable replacement to me for this eventual use case.